Library/Homebrew/language/node.rb

```diff
@@ -4,15 +4,27 @@ def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache\n"
     end
 
-    def self.pack_for_installation
+    def self.pack_for_installation(prepublish_requires_deps, pack_skip_prepublish)
```

Comment:
I think they should be keyword or hash arguments; taking two booleans here means the method calls are a bit opaque.